test/client/fixtures/sandbox/shadow-ui-test.js

```diff
@@ -438,6 +439,103 @@ test('querySelectorAll', function () {
     strictEqual(elems[0].id, 'pageElem');
 });
 
+module('ui stylesheet', {
```

Comment:
AFAIK we don't define modules this way in our client tests. Use
`module('ui stylesheet');` instead